### DISCUSSION TITLE
Erro na requisição usando RestRequest4D

### DISCUSSION THREAD

**Discussion by AdailtonAlves** [viniciussanchez/RESTRequest4Delphi#186] [General]
Ola, estou tendo um erro que começou do nada, estava tudo funcionando e começou reclamar do DataSetAdapter ao montar a requisição, usando o RestRequest4D. Segue o print do trecho onde aponta o "problema"
![Captura de tela 2023-12-27 121610](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/61528599/a7a56a16-6ba0-4ac5-8c02-31ee402c74d8)
O que pode ter acontecido? Estou sem saber o que fazer. Ja baixei os componentes novamente e refiz as referências, mas nada mudou. Pode por favor me ajudar?

**Comment by viniciussanchez** [comment]
Boa tarde... a partir da versão 4, trabalhamos com os adapters: https://github.com/viniciussanchez/RESTRequest4Delphi/issues/131

![image](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/16382981/35c64f7c-5d94-4438-a348-152e5e19ec5c)

![image](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/16382981/80af5073-c05d-4a19-8d64-430efd54012d)

https://github.com/viniciussanchez/dataset-serialize-adapter-restrequest4delphi

Obs: se não quiser mudar, pode usar versões inferiores a 4.0.0
